yarn.lock

```diff
@@ -961,6 +961,16 @@
     lodash "^4.17.11"
     to-fast-properties "^2.0.0"
 
+"@clayui/navigation@^3.0.0-alpha.1":
```

Comment:
This probably shouldn't be here, where are we using it? Maybe it's at `next.clayui.com` that was downloading this dependency.